src/app/docs/box/box-docs.component.html

```diff
@@ -29,14 +29,162 @@
     ></div>
```

Comment:
What do we need to do to get the demo to work in default theme?

And what portions of this content do we need to hide in default theme? The links in the Related information section point to the modern theme site, so I followed that pattern on my recommendations for additional links. But those links won't work for external users. I think we should use the directive that filters content by theme to either hide this content in the default site or to make it work for both sites.

Reply:
Box itself needs styles added for default theme.

Reply:
Does that mean we should hide the box component from the default theme site? Right now, we have the component documented on the default theme site (https://developer.blackbaud.com/skyux-theme/docs/box?docs-active-tab=design), but the demo just has the demo border and doesn't have the input box defined within it. And most of the links you've added are to modern theme pages, which will work fine for Blackbaud users who are logged in on the default site but will be busted links for any third-party developers on the site.

Reply:
yeah for the time being we should exclude it from the component list for the default theme site (assuming that is possible)

Reply:
I assumed we could hide components from the default site because there's a property in the docs service where we indicate which site to display the component docs in. But the box component is not set up to display in the default site, so we either have a bug or need to change something in another place too. I pinged Alex to find out what's up since he's the developer most familiar with the modern theme site.